driver-embedded/src/main/com/mongodb/embedded/client/MongoEmbeddedSettings.java

```diff
@@ -19,6 +19,7 @@
 
 import com.mongodb.annotations.Immutable;
 import com.mongodb.annotations.NotThreadSafe;
+import com.mongodb.embedded.capi.LogLevel;
```

Comment:
This look like the only place where users of driver-embedded depend on the public API in capi? I think it's worth it to hide that and wrap CAPI's LogLevel enum with our own so that there is no direct dependency.

Reply:
Good idea - done.